### ISSUE TITLE
matplotlib cannot be interactive

### ISSUE THREAD

**Issue by sylvain-bougnoux** [Microsoft/PTVS#857]
import matplotlib.pyplot as plt
plt.ion()  # this create the crash
plt.plot([1,2])

A window appears (gui=qt) but gets freeze without any plot. The window must be killed (closed) which crash the REPL.However the prompt was back. 
Without plt.ion() it works after calling plt.show(), but the figure cannot be changed dynamically.

Thanks

python 2.7, VS2013pro, PTVS 2.2, matplotlib 1.4.3

**Comment by zooba** [comment]
We recommend [configuring IPython](https://github.com/Microsoft/PTVS/wiki/Using-IPython-with-PTVS) (our doc page) for this. UI and REPLs generally don't work well together unless it's been explicitly designed into both parts (in this case, the UI does not support our REPL).

**Comment by sylvain-bougnoux** [comment]
Thanks for answering.
My original question was related to python interactive, not ipython interactive.
However I did some test, here are the results.

in "python interactive" as described above plt.ion() corrupts the REPL.
in "ipython interactive" there is no need to configure "Interactive Mode:" as "IPython w/o Pylab" as claim in the doc to embed a plot/image within an external window. One just has to:
In [1]: %matplotlib tk
In [2]: plt.plot([1,2]) # pops a window, and give the prompt back

plt.ion() is not needed, it is by default.

PS: In fact I found it is impossible to use "IPython w/o Pylab". Once click "OK", and call "Configure interactive window", this option is back to "IPython".

Regards.
